app/src/androidTest/java/com/fstyle/structure_android/ExampleInstrumentedTest.java

```diff
@@ -7,7 +7,7 @@
 import org.junit.Test;
 import org.junit.runner.RunWith;
 
-import static org.junit.Assert.*;
+import static org.junit.Assert;
```

Comment:
ERROR: Wrong order for org.junit.Assert import. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports